src/MockClassifier.Api/Interfaces/IClassifierService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace MockClassifier.Api.Interfaces
+{
+public interface IClassifierService
```

Comment:
Ideally, interfaces should have XML based comments: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/